src/components/Chat/AnonymousInputMessage.tsx

```diff
@@ -66,23 +73,137 @@ const styles = StyleSheet.create({
 });
 
 export interface AnonymousInputMessageProps {
-  onSendButtonClicked: (message: string) => void;
+  onSendButtonClicked: (message: string, attachments: any) => void;
   type: 'SIGNED' | 'ANONYMOUS';
 }
 
 const AnonymousInputMessage = ({onSendButtonClicked, type}: AnonymousInputMessageProps) => {
   const refEmoji = React.useRef(null);
+  const refAttachment = React.useRef(null);
   const [text, setText] = React.useState('');
+  const [isLoadingUploadImageMedia, setIsLoadingUploadImageMedia] = React.useState(false);
+  const [isLoadingUploadImageCamera, setIsLoadingUploadImageCamera] = React.useState(false);
+
+  const handleUploadMedia = async (paths) => {
+    setIsLoadingUploadImageMedia(true);
+
+    const resultUrls = paths.map((path) => ({
+      type: 'image',
+      asset_url: path,
+      thumb_url: path,
+      myCustomField: 'image'
+    }));
+    console.warn('resultUrls', resultUrls);
+    onSendButtonClicked(' ', resultUrls);
+    setIsLoadingUploadImageMedia(false);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This function `handleUploadMedia` is uploading images without any error handling. If the upload fails for some reason, the user will not be notified and the application may behave unexpectedly. Consider adding a try-catch block to handle potential errors.



<!-- This is an auto-generated comment by OSS CodeRabbit -->